datafusion/sqllogictest/test_files/aggregate.slt

```diff
@@ -1826,7 +1826,7 @@ from values
   (interval '2 month 15 days'),
   (interval '-2 month')
 ----
-Interval(MonthDayNano) 0 years -2 mons 0 days 0 hours 0 mins 0.000000000 secs 0 years 2 mons 15 days 0 hours 0 mins 0.000000000 secs
+Interval(MonthDayNano) -2 mons 2 mons 15 days
```

Comment:
This display difference is due to https://github.com/apache/arrow-rs/pull/6006 from @Rachelint to make the display more readable ❤️